_pytest/pytester.py

```diff
@@ -908,8 +908,11 @@ def popen(self, cmdargs, stdout, stderr, **kw):
         env['PYTHONPATH'] = os.pathsep.join(filter(None, [
             str(os.getcwd()), env.get('PYTHONPATH', '')]))
         kw['env'] = env
-        return subprocess.Popen(cmdargs,
-                                stdout=stdout, stderr=stderr, **kw)
+
+        popen = subprocess.Popen(cmdargs, stdin=subprocess.PIPE, stdout=stdout, stderr=stderr, **kw)
```

Comment:
nitpick, please add a comment to switch to DEVNULL once we drop python2.6 support